target/scripts/startup/setup-stack.sh

```diff
@@ -634,51 +634,52 @@ function _setup_SRS
 
 function _setup_dkim_dmarc
 {
-  if [[ ${ENABLE_OPENDMARC} -eq 1 ]]
+  if [[ ${ENABLE_OPENDKIM} -eq 1 ]]
```

Comment:
it works for our servers (opendkim + opendmarc enabled) but the `return 0` is lost here.

which syntactically was useless in the current state of the code (after lot's of refactoring) but made me think why it was there in the first place.

should opendmarc be enabled without opendkim (and policyd-spf) or should there be at least a warning that all mails for domains with DMARC enable are incorrectly rejected if you run opendmarc without policyd-spf or without opendkim?

for a [RFC 7489](https://www.rfc-editor.org/rfc/rfc7489#page-24) compliant mail receiver setup if opendmarc is enabled (ENABLE_OPENDMARC), policyd-spf (always enabled) and opendkim (ENABLE_OPENDKIM) must be enabled.

Reply:
> should opendmarc be enabled without opendkim (and policyd-spf) or should there be at least a warning that all mails for domains with DMARC enable are incorrectly rejected if you run opendmarc without policyd-spf or without opendkim?

There is definitely room for improvments. This PR only aims to fix the current regression. At the moment, I don't have much spare time. So looking into it, will take same weeks.